include/stlink.h

```diff
@@ -58,6 +58,7 @@ extern "C" {
     // Constant STM32 memory map figures
 #define STM32_FLASH_BASE 0x08000000
 #define STM32_SRAM_BASE 0x20000000
+#define STM32_G0_OPTION_BYTES_BASE ((uint32_t)0x1FFF7800)
```

Comment:
we should create an include/stlink/stm32.h header file (and new folder). and put it in there also with the ones on line 59 + 60. as the stlink.h is "the entry point" for the library it is good to not pollute it with lowlevel register definitions.

Reply:
created